io.github.twisterarmy.twister.json

```diff
@@ -0,0 +1,95 @@
+{
+    "app-id": "io.github.twisterarmy.twister",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "twister",
+    "finish-args": [
+        "--share=network",
+        "--persist=.twister"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/share/cmake",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=/app",
+                "./b2 install -j $FLATPAK_BUILDER_N_JOBS"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://archives.boost.io/release/1.83.0/source/boost_1_83_0.tar.gz",
+                    "sha256": "c0685b68dd44cc46574cce86c4e17c0f611b15e195be9848dfd0769a0a207628"
+                }
+            ]
+        },
+        {
+            "name": "libdb",
+            "buildsystem": "simple",
```

Comment:
this still should be `buildsystem: autotools`

Reply:
also independently it probably need a shell source like this:

https://github.com/flathub/org.gimp.GIMP/blob/master/org.gimp.GIMP.json#L344

to build on aarch64 (old tarball with old autoconf)